web-client/src/views/StartCaseInternal/CalculatePenaltiesModal.tsx

```diff
@@ -48,7 +53,14 @@ export const CalculatePenaltiesModal = connect(
         {penalties &&
           penalties.map((penalty, index) => (
             // eslint-disable-next-line react/no-array-index-key
-            <div className="margin-top-3" key={index}>
+            // <div className="margin-top-3" key={index}>
```

Comment:
Nit: little clean up here for comments and warning on naming the key as the array index

Reply:
Thanks! We switched the key to the only unique value available to us on the penalty object.